GOVERNANCE.md

```diff
@@ -0,0 +1,246 @@
+* From the algorithms or techniques that meet the above criteria, only those which fit well within the current framework and API of sktime are accepted. For extending current frameworks and API, see the [process for major changes]().
+* The contributor should support the importance of the proposed addition with research papers and/or implementations in other similar packages, demonstrate its usefulness via common use-cases/applications and corroborate performance improvements, if any, with benchmarks and/or plots. It is expected that the proposed algorithm should outperform the methods that are already implemented in sktime in at least some areas.
+* We strive to consolidate existing functionality if helps to improve the usability and maintainability of the project. For example, when there are multiple techniques for the same purpose, we prefer to combine them into a single class and make case distinctions based on hyper-parameters.
+
+Note that your implementation need not be in sktime to be used together with sktime tools. You can implement your favorite algorithm in a sktime
+compatible way in one of [our companion repositories](https://github.com/sktime) on GitHub. We will be happy to list it under [related software](https://github.com/alan-turing-institute/sktime/wiki/related-software).
+
+If algorithms require major dependencies, we encourage to create a separate companion repository. For example, for deep learning techniques based on TensorFlow and Keras, we have [sktime-dl](https://github.com/sktime/sktime-dl). For smaller dependencies which are limited to a few files, we encourage to use soft dependencies, which are only required for particular modules, but not for most of sktime's functionality and not for installing sktime.
+
+## Acknowledging contributions
+
+sktime is collaboratively developed by its diverse community of developers, users, educators, and other stakeholders. We value all kinds of contributions and are committed to recognising each of them fairly.
+
+We follow the [all-contributors](https://allcontributors.org) specification to recognise all contributors, including those that don't contribute code. Please see [our list of all contributors](https://github.com/alan-turing-institute/sktime/blob/master/CONTRIBUTORS.md).
+
+If you think, we've missed anything, please let us know or open a PR with the appropriate changes to [`sktime/.all-contributorsrc`](https://github.com/alan-turing-institute/sktime/blob/master/.all-contributorsrc).
+
+Note that contributors do not own their contributions. sktime is an open-source project, and all code is contributed under [our open-source license](https://github.com/alan-turing-institute/sktime/blob/master/LICENSE). 
```

Comment:
might be important to say: all code contributors acknowledge they have all the rights to the code they contribute to make it available under this license.

Reply:
possible license problem might arise if someone takes utility functions or core methodology code from a stricter package, e.g., subject to copyleft or copyright